### PR TITLE
Update punctuation.md, add the Date format page

### PR DESCRIPTION
**Pull request by ana-sorokina**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: website/docs/content/punctuation/punctuation.md [hunk 1]
```diff
@@ -44,24 +44,24 @@ title: Punctuation and special symbols
...
         <th>Do</th>
     </tr>
     <tr>
         <td>
-            It makes us sound like Semrush &mdash; regardless of who the person behind the communication&nbsp;is.
+            It makes us sound like Semrush &mdash; regardless<br />of who the personbehind the communication is.
         </td>
         <td>
-            It makes us sound like Semrush&mdash;regardless of who the person behind the communication&nbsp;is.
+            It makes us sound like Semrush&mdash;regardless<br />of who the personbehind the communication is.
```
Comment (ana-sorokina) [inline website/docs/content/punctuation/punctuation.md]: I know about the missing space, but I'm not sure I'm able to add a quick fix to the current PR

File: website/docs/content/punctuation/punctuation.md [hunk 2]
```diff
@@ -162,8 +342,37 @@ title: Punctuation and special symbols
...
+        <td>
+            ☑️ Remind later.
+        </td>
+        <td>
+            ☑️ Remind later
+        </td>
+    </tr>
+    <tr>
+        <td>
+            ???
```
Comment (ana-sorokina) [inline website/docs/content/punctuation/punctuation.md]: I'll need help adding an example in a form of an image. But it's small, you can barely see anything, so maybe you'll have a better idea

File: website/docs/content/punctuation/punctuation.md [hunk 3]
```diff
@@ -44,24 +44,24 @@ title: Punctuation and special symbols
...
 <table class="dosdonts">
     <tr>
-        <th>Don't</th>
+        <th>Don’t</th>
         <th>Do</th>
     </tr>
     <tr>
         <td>
-            It makes us sound like Semrush &mdash; regardless of who the person behind the communication&nbsp;is.
+            It makes us sound like Semrush &mdash; regardless<br />of who the personbehind the communication is.
```
Comment (sheila-semrush) [inline website/docs/content/punctuation/punctuation.md]:
```suggestion
            It makes us sound like Semrush &mdash; regardless<br />of who the person behind the communication is.
```
@ana-sorokina you should see something like "accept suggestion" here, try it

File: website/docs/content/punctuation/punctuation.md (hunk 1 above)
Comment (sheila-semrush) [inline website/docs/content/punctuation/punctuation.md]:
```suggestion
            It makes us sound like Semrush&mdash;regardless<br />of who the person behind the communication is.
```

**Review by sheila-semrush — APPROVED** [review]

File: website/docs/content/punctuation/punctuation.md (hunk 3 above)
Comment (sheila-semrush) [reply website/docs/content/punctuation/punctuation.md]: ahh it's marked as outdated. Ok, I'll fix it on our side.

File: website/docs/content/punctuation/punctuation.md (hunk 1 above)
Comment (sheila-semrush) [reply website/docs/content/punctuation/punctuation.md]: same as above

File: website/docs/content/punctuation/punctuation.md (hunk 2 above)
Comment (sheila-semrush) [reply website/docs/content/punctuation/punctuation.md]: I'll look at it in a moment

**Pull request merged**